GUISupport/Qt/QVTKInteractorAdapter.cxx

```diff
@@ -77,6 +78,30 @@ void QVTKInteractorAdapter::SetDevicePixelRatio(int ratio, vtkRenderWindowIntera
   }
 }
 
+int QVTKInteractorAdapter::UpdateTouchPointIdMap(int qtPointId)
+{
+  if (TouchPointIdMap.contains(qtPointId))
+  {
+    return GetPointerIndex(qtPointId);
+  }
+  int pointerIndex = 0;
+  for (; pointerIndex <= VTKI_MAX_POINTERS; ++pointerIndex)
+  {
+    if (!TouchPointIdMap.values().contains(pointerIndex))
+    {
+      TouchPointIdMap.insert(qtPointId, pointerIndex);
+      break;
+    }
+    pointerIndex++;
+  }
+  return pointerIndex <= VTKI_MAX_POINTERS ? pointerIndex : -1;
```

Comment:
test is not needed (pointerIndex is ALWAYS <= VTKI_MAX_POINTERS)

Reply:
It's a mistake, it should be <.
But pointerIndex will not be < VTKI_MAX_POINTERS if the map is full.